lib/model/text_answer_model.dart

```diff
@@ -0,0 +1,9 @@
+class TextAnswerModel {
+  final String answerId;
+  final String answerText;
```

Comment:
Can we make `answerText` to be nullable?  Then we don't need to create separate listeners for `onSelectedIndexedAnswers` & `onUpdatedTextAnswers`. The code will be a lot shorter in `form_screen.dart` 💭 

Please let me know if I missed something 🙏